src/wirecloud/platform/tests/selenium_tests.py

```diff
@@ -115,6 +115,20 @@ def test_remove_widget_from_workspace(self):
         self.login(username="user_with_workspaces", next="/user_with_workspaces/Workspace")
         self.find_widget(title="Test 1").remove()
 
+    def test_remove_widget_from_workspace_with_behaviours(self):
```

Comment:
Remove this selenium test and provide a unit test based on the current "test_iwidget_removed" test provided by the "WiringTestCase" class (`wirecloud/platform/wiring/tests.py`).